src/cookies.zig

```diff
@@ -25,24 +25,62 @@ pub const Attributes = struct {
         none,
     };
 
+    /// vec must be large enough for the largest cookie (10)
+    pub fn writeVec(a: Attributes, vec: []std.posix.iovec_const) !usize {
+        var used: usize = 0;
+        if (a.domain) |d| {
+            vec[used] = .{ .base = "; Domain=", .len = 9 };
+            used += 1;
+            vec[used] = .{ .base = d.ptr, .len = d.len };
+            used += 1;
+        }
+        if (a.path) |p| {
+            vec[used] = .{ .base = "; Path=", .len = 7 };
+            used += 1;
+            vec[used] = .{ .base = p.ptr, .len = p.len };
+            used += 1;
+        }
+        if (a.max_age) |_| {
```

Comment:
Should we just get rid of the max age attribute?

Reply:
Tempting, I would be nice to drop it, but I'd just add it back once someone opened an issue for it being missing.